CDN link not working after update

Cdn link not working

https://unpkg.com/buefy@0.9.24/dist/buefy.min.js
Cannot  find "/dist/buefy.min.js" in buefy@0.9.24


**Buefy** version: [0.9.24]]

Also the NPM package is not working after the update 😕

this code
```js
import { ModalProgrammatic } from 'buefy'
```

throws the following error after updating to 0.9.24
```
Module not found: Error: Can't resolve 'buefy' in '/..../src/pages/backend'
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -7,7 +7,7 @@ jobs:
   build:
     docker:
       # specify the version you desire here
-      - image: circleci/node:8.9-browsers
+      - image: cimg/node:18.18.0-browsers
 
     working_directory: ~/repo
 

--- a/package.json
+++ b/package.json
@@ -50,7 +50,8 @@
     "build:docs": "node build/build.js && copyfiles --flat netlify.toml dist-docs",
     "build": "npm run build:lib && npm run build:docs",
     "cov": "./node_modules/codcov/bin/codcov",
-    "vetur": "node -r esm build/vetur.js"
+    "vetur": "node -r esm build/vetur.js",
+    "prepare": "npm run build:lib"
   },
   "keywords": [
     "bulma",

--- a/src/components/datepicker/Datepicker.spec.js
+++ b/src/components/datepicker/Datepicker.spec.js
@@ -264,10 +264,17 @@ describe('BDatepicker', () => {
     })
 
     describe('#dateFormatter', () => {
+        beforeEach(() => {
+            // uses en-US locale to make outputs predictable
+            wrapper.setProps({
+                locale: 'en-US'
+            })
+        })
+
         it('should add one to month since month in dates starts from 0', () => {
             const dateToFormat = new Date(2019, 3, 1)
             const formattedDate = wrapper.vm.dateFormatter(dateToFormat, wrapper.vm)
-            expect(formattedDate).toEqual('2019-4-1')
+            expect(formattedDate).toEqual('4/1/2019')
         })
 
         it('should format based on 2-digit numeric locale date with type === month', () => {
@@ -276,7 +283,7 @@ describe('BDatepicker', () => {
             })
             const dateToFormat = new Date(2019, 3, 1)
             const formattedDate = wrapper.vm.dateFormatter(dateToFormat, wrapper.vm)
-            expect(formattedDate).toEqual('2019-04')
+            expect(formattedDate).toEqual('4/2019')
         })
 
         it('should format a range of dates passed via array', () => {
@@ -285,7 +292,7 @@ describe('BDatepicker', () => {
                 new Date(2019, 3, 3)
             ]
             const formattedDate = wrapper.vm.dateFormatter(dateToFormat, wrapper.vm)
-            expect(formattedDate).toEqual('2019-4-1 - 2019-4-3')
+            expect(formattedDate).toEqual('4/1/2019 - 4/3/2019')
         })
 
         describe('multiple', () => {
@@ -305,7 +312,7 @@ describe('BDatepicker', () => {
                     new Date(2019, 3, 3)
                 ]
                 const formattedDate = wrapper.vm.dateFormatter(dateToFormat, wrapper.vm)
-                expect(formattedDate).toEqual('2019-4-1, 2019-4-13, 2019-4-3')
+                expect(formattedDate).toEqual('4/1/2019, 4/13/2019, 4/3/2019')
             })
 
             it('react accordingly when setting computedValue', () => {

--- a/src/components/datepicker/DatepickerMonth.spec.js
+++ b/src/components/datepicker/DatepickerMonth.spec.js
@@ -5,12 +5,10 @@ import config, {setOptions} from '@utils/config'
 
 let wrapper
 
+// it used to create a Date in UTC but should use the local timezone,
+// because DatepickerMonth deals with the local timezone
 const newDate = (y, m, d) => {
-    const date = new Date(Date.UTC(y, m, d))
-    date.getDate = jest.fn(() => date.getUTCDate())
-    date.getMonth = jest.fn(() => date.getUTCMonth())
-    date.getFullYear = jest.fn(() => date.getUTCFullYear())
-    return date
+    return new Date(y, m, d)
 }
 
 const thisMonth = newDate(2020, 4, 15).getUTCMonth()

--- a/src/components/datetimepicker/Datetimepicker.spec.js
+++ b/src/components/datetimepicker/Datetimepicker.spec.js
@@ -69,6 +69,7 @@ describe('Datetimepicker', () => {
         const datetimeToFormat = new Date(2019, 9, 1, 8, 30, 0, 0)
         wrapper = mount(Datetimepicker, {
             propsData: {
+                locale: 'en-US', // makes outputs predictable
                 value: datetimeToFormat
             },
             stubs: {
@@ -77,7 +78,7 @@ describe('Datetimepicker', () => {
         })
         const datepicker = wrapper.find({ ref: 'datepicker' })
 
-        expect(datepicker.vm.formattedValue).toEqual('2019-10-1 08:30')
+        expect(datepicker.vm.formattedValue).toEqual('10/1/2019, 8:30 AM')
 
         wrapper.setProps({
             datetimeFormatter: (date) => `${date.getFullYear()}`
@@ -88,6 +89,7 @@ describe('Datetimepicker', () => {
     it('should format date time with seconds', () => {
         wrapper = mount(Datetimepicker, {
             propsData: {
+                locale: 'en-US', // makes outputs predictable
                 timepicker: {
                     enableSeconds: true
                 }
@@ -98,13 +100,14 @@ describe('Datetimepicker', () => {
         })
         const datetimeToFormat = new Date(2019, 9, 1, 8, 30, 0, 0)
         const formattedDatetime = wrapper.vm.defaultDatetimeFormatter(datetimeToFormat)
-        expect(formattedDatetime).toEqual('2019-10-1 08:30:00')
+        expect(formattedDatetime).toEqual('10/1/2019, 8:30:00 AM')
     })
 
     it('should format date time according init value', async () => {
         const date = new Date(2019, 9, 1, 8, 30, 0, 0)
         wrapper = mount(Datetimepicker, {
             propsData: {
+                locale: 'en-US', // makes outputs predictable
                 value: date
             },
             stubs: {
@@ -113,11 +116,14 @@ describe('Datetimepicker', () => {
             sync: false
         })
         await wrapper.vm.$nextTick()
-        expect(wrapper.find('input').element.value).toEqual('2019-10-1 08:30')
+        expect(wrapper.find('input').element.value).toEqual('10/1/2019, 8:30 AM')
     })
 
     it('should parse date time', async () => {
         wrapper = mount(Datetimepicker, {
+            propsData: {
+                locale: 'en-US' // makes outputs predictable
+            },
             stubs: {
                 transition: false
             }

--- a/src/components/numberinput/Numberinput.spec.js
+++ b/src/components/numberinput/Numberinput.spec.js
@@ -170,7 +170,18 @@ describe('BNumberinput', () => {
 
     describe('Rendered (shallow)', () => {
         beforeEach(() => {
-            wrapper = shallowMount(BNumberinput)
+            wrapper = shallowMount(BNumberinput, {
+                stubs: {
+                    // to suppress error messages printed when BNumberinput
+                    // tries to call checkHtml5Validity of the input
+                    'b-input': {
+                        template: '<div></div>',
+                        methods: {
+                            checkHtml5Validity: () => true
+                        }
+                    }
+                }
+            })
         })
 
         it('manage prop types (number / string)', () => {

--- a/src/components/numberinput/Numberinput.spec.js
+++ b/src/components/numberinput/Numberinput.spec.js
@@ -3,10 +3,6 @@ import BNumberinput from '@components/numberinput/Numberinput'
 
 let wrapper
 
-const wait = (time) => {
-    return new Promise((resolve) => setTimeout(resolve, time))
-}
-
 describe('BNumberinput', () => {
     describe('Rendered', () => {
         beforeEach(() => {
@@ -35,28 +31,46 @@ describe('BNumberinput', () => {
         })
 
         it('increments/decrements on long pressing exponentially', async () => {
+            // we should not depend on a real timer
+            // otherwise the results will depend on the machine
+            jest.useFakeTimers()
+
             wrapper.setProps({exponential: true, value: 1, step: 1})
 
             wrapper.find('.control.plus button').trigger('mousedown')
+            expect(wrapper.vm.computedValue).toBe(2)
 
-            await wait(1500)
+            for (let n = 1; n <= 10; ++n) {
+                // while Jest's fake setTimeout truncates the delay to an exact
+                // ms when it schedules the callback,
+                // jest.advanceTimersByTime floors the value and
+                // accumulates the remainder.
+                // so should be floored to prevent the accumulated remainder
+                // from triggering an extra callback
+                jest.advanceTimersByTime(Math.floor(250 / n))
+                expect(wrapper.vm.computedValue).toBe(n + 2)
+            }
 
             wrapper.find('.control.plus').trigger('mouseup')
+            jest.runAllTimers()
             await wrapper.vm.$nextTick()
-
-            expect(wrapper.vm.computedValue).toBeGreaterThan(1)
-            expect(wrapper.vm.computedValue).toBeLessThan(150)
+            expect(wrapper.vm.computedValue).toBe(12)
 
             wrapper.setProps({exponential: 3, value: 0, step: 1})
 
             wrapper.find('.control.plus button').trigger('mousedown')
+            expect(wrapper.vm.computedValue).toBe(1)
 
-            await wait(1000)
+            for (let n = 1; n <= 20; ++n) {
+                // see above for `floor`ing
+                jest.advanceTimersByTime(Math.floor(250 / (n * 3)))
+                expect(wrapper.vm.computedValue).toBe(n + 1)
+            }
 
             wrapper.find('.control.plus').trigger('mouseup')
-
-            expect(wrapper.vm.computedValue).toBeGreaterThan(0)
-            expect(wrapper.vm.computedValue).toBeLessThan(600)
+            jest.runAllTimers()
+            await wrapper.vm.$nextTick()
+            expect(wrapper.vm.computedValue).toBe(21)
         })
 
         it('increments/decrements on long pressing', async () => {

--- a/src/components/timepicker/Timepicker.spec.js
+++ b/src/components/timepicker/Timepicker.spec.js
@@ -5,7 +5,13 @@ let wrapper
 
 describe('BTimepicker', () => {
     beforeEach(() => {
-        wrapper = shallowMount(BTimepicker)
+        wrapper = shallowMount(BTimepicker, {
+            propsData: {
+                // fixes the locale, and hour format to make outputs predictable
+                locale: 'en-US',
+                hourFormat: '24'
+            }
+        })
     })
 
     it('is called', () => {

